### PR TITLE
Add snippet for formatSeconds

### PR DESCRIPTION
**Pull request by shanyuhai123**

### CODE REVIEW

**Review by lodenrogue — APPROVED** [review]
Nice

**Review by SandhuSukh — APPROVED** [review]

**Comment by SandhuSukh** [comment]
Good

**Comment by Charle72** [comment]
[inthecode]()

**Review by bl-ue — APPROVED** [review]

**Review by hugo105 — APPROVED** [review]

**Review by hugo105 — APPROVED** [review]

**Review by hugo105 — APPROVED** [review]

**Review by hugo105 — APPROVED** [review]

**Review by hugo105 — APPROVED** [review]

**Comment by kofilevelz** [comment]
Thanks 🙏🏽

File: snippets/formatSeconds.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (MJjames43) [inline snippets/formatSeconds.md:1]: snippets/formatSeconds.md

**Review by MJjames43 — APPROVED** [review]

**Review by Chalarangelo — APPROVED** [review]
Thanks for contributing and sorry for the random spam. I made some changes to make the snippet a little more robust.

**Pull request merged**